fvm/evm/emulator/state/base.go

```diff
@@ -421,7 +421,12 @@ func (v *BaseView) getSlot(sk types.SlotAddress) (gethCommon.Hash, error) {
 		return value, nil
 	}
 
-	col, err := v.getSlotCollection(sk.Address)
+	acc, err := v.getAccount(sk.Address)
+	if err != nil || acc == nil || len(acc.CollectionID) == 0 {
```

Comment:
wouldn't it be better to return an error if one exists and only if acc is nil or doesn't have collection return empty hash? this way error getting an account isn't swallowed that might be unrelated to the existance of the account

Reply:
nice find, it was supposed to return the error on the return statement as well but I forgot, I split them for better readability.